lib/Newsletter/Sending/ScheduledTasksRepository.php

```diff
@@ -46,6 +46,23 @@ public function findByNewsletterAndSubscriberId(NewsletterEntity $newsletter, in
       ->getResult();
   }
 
+  /**
+   * @param string[] $types
+   * @return ScheduledTaskEntity|null
+   */
+  public function findScheduledOrRunningTask(array $types = []): ?ScheduledTaskEntity {
```

Comment:
This method returns an entity or null. Why did you decide to add a parameter of type array here? I think that when somebody calls this method with two types, it will end with an error.